### DISCUSSION TITLE
ts | v-infinite-scroll onLoad  Type Error

### DISCUSSION THREAD

**Discussion by q962** [vuetifyjs/vuetify#20233] [Q&A]
```ts
async function load({done }) {}
```

```
error TS7031: Binding element 'done' implicitly has an 'any' type.

33 async function load({ done }) {

```

Cannot compile, what is the reason?

**Comment by melondoto** [comment]
[SOLVED]
you can code like this:
`async function load({done }: { done: (status: 'ok' | 'empty' | 'loading' | 'error') => void }) {}`

OR:
`async function load({done }: { done: any) {}`
